src/foam/nanos/menu/SubMenu.js

```diff
@@ -17,8 +17,7 @@ foam.CLASS({
     },
 
     function launch(X, menu, parent) {
-      var view = this.createView(X, menu, parent);
-      view.open();
+      // noop
```

Comment:
Will this break the old style menus? Do we plan on supporting both styles?

Reply:
Yes, and I don't know. Probably safer to go with @zahnno's fix.